desktop/renderer/index.desktop.js

```diff
@@ -9,6 +9,7 @@ import configureStore from '../../react-native/react/store/configure-store'
 import Nav from '../../react-native/react/nav'
 import { DevTools, DebugPanel, LogMonitor } from 'redux-devtools/lib/react'
 import injectTapEventPlugin from 'react-tap-event-plugin'
+import { enableNotifications, bindNotifications, unbindNotifications} from '../../react-native/react/native/notifications'
```

Comment:
do we need to reach into react-native to get this? doesn't desktop have react directly?